src/bin/Cicero.pl

```diff
@@ -174,27 +168,21 @@
 system("touch $out_dir/unfiltered.internal.txt"); 
 
 # the softclip file is sorted, so no need to re-sort it
-#print  "sclip_file: $sclip_file\n";
```

Comment:
Is there a debug option that can be used and still print some of this stuff? I feel like for research operations this kind of output can help us to diagnose hard to solve problems.

Reply:
Ah - just realized that a lot of this is just removing code that is already commented out so you aren't actually reducing verbosity just cleaning out dead code.

Reply:
Yeah, there is a debug option (though the global option is overridden in each function) that prints debugging output. I was just focusing on removing things that were already commented out.